src/components/StudyHeader/StudyHeader.jsx

```diff
@@ -11,40 +12,35 @@ const StudyHeader = ({
   name: studyName,
   className,
 }) => {
-  const studyHeaderClasses = classes('StudyHeader', 'BodyContent', className);
+  const studyHeaderClass = classes('bg-lightGrey', className);
+  const topTextClass = classes('topText');
   return (
-    <div className="bg-lightGrey">
-      <header className={studyHeaderClasses}>
-        <div>
-          <h3 className="mt-1 text-base">
-            <span className="font-body font-bold pr-4 text-darkGrey" id="kfId">
-              {kfId}
-            </span>
-            <CopyButton text={kfId} />
-            <span className="study-modified-date pl-20 font-light">
-              <small>
-                last updated: <TimeAgo date={modifiedAt} />
-              </small>
-            </span>
-          </h3>
-          <h2 className="mt-4 font-light text-blue text-4xl">
-            {shortName || studyName}
-          </h2>
+    <div className={studyHeaderClass}>
```

Comment:
Is it possible to remove this div and put the class name on the GridContainer to avoid another <div>?